src/Smart.Design.Razor/TagHelpers/Titles/PageTitle/PageTitleTagHelper.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Smart.Design.Razor.TagHelpers.Constants;
+
+
+namespace Smart.Design.Razor.TagHelpers.Titles.PageTitle;
+
+[HtmlTargetElement(TagNames.PageTitleTagName)]
+public class PageTitleTagHelper : TagHelper
+{
+    private readonly IPageTitleHtmlGenerator _pageTitleGenerator;
+
+    /// <summary>
+    /// Title of the page.
+    /// </summary>
+    public string Title { get; set; }
+
+    /// <summary>
+    /// Creates a new <see cref="PageTitleHtmlGenerator"/>.
+    /// </summary>
+    /// <param name="pageTitleHtmlGenerator">The service that generates Smart Design H1 page Title HTML.</param>
+    public PageTitleTagHelper(IPageTitleHtmlGenerator pageTitleHtmlGenerator)
+    {
+        _pageTitleGenerator = pageTitleHtmlGenerator;
+    }
+
+    /// <inheritdoc />
+    public override void Process(TagHelperContext context, TagHelperOutput output)
+    {
```

Comment:
Might be a good COP back topic, but since the component doesn't make much sense if there are not titles (and that might be involuntary),  we could want to add that:
```suggestion
    {
        if (string.IsNullOrWhiteSpace(Title))
        {
            throw new MissingFieldException(nameof(PageTitleTagHelper), nameof(Title));
        }
```